contracts/security-token/ERC865.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.4.18;
+
+import "./CheckpointToken.sol";
+
+/* Largely copied from https://github.com/OpenZeppelin/openzeppelin-solidity/pull/741/files */
+
+contract ERC865 is CheckpointToken {
+  mapping(bytes => bool) signatures;
+
+  event TransferPreSigned(address indexed from, address indexed to, address indexed delegate, uint256 amount, uint256 fee);
+  event Debug(address from, bytes32 hash);
+
+  /**
+    * @notice Submit a presigned transfer
+    * @param _signature bytes The signature, issued by the owner.
+    * @param _to address The address which you want to transfer to.
+    * @param _value uint256 The amount of tokens to be transferred.
+    * @param _fee uint256 The amount of tokens paid to msg.sender, by the owner.
```

Comment:
Also let me get it clarified here: the beneficial sender (who signs the message) can set the fee in tokens? The tokens are always the underlying token contract?

Reply:
Added clarification :slightly_smiling_face: